new/LIT14519Winds.xml

```diff
@@ -0,0 +1,68 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT14523Winds" xml:lang="en" type="work">
```

Comment:
I understand, that LIT14519Winds is identical to LIT7123Winds and will be deleted in another pull request, correct? Otherwise the xml:id must be changed to be identical to the file name.

Reply:
They can be deleted in this PR as well, so that only LIT7122Diagram 
 and LIT7123Winds are extant. Now they should have been correctly deleted, isn't it? Thank you :)

Reply:
yes they are deleted on the backend already, and merging these files would recreate the numbering error, thank you for removing them and updating the xml:id